includes/wikia/models/DesignSystemFooterModel.class.php

```diff
@@ -382,18 +382,23 @@ private function getVerticalData() {
 			$link = 'http://fandom.wikia.com/' . $vertical;
 		}
 
-		if ( !in_array( $vertical, $this->allowedVerticals ) ) {
+		if ( is_null( $vertical ) ) {
 			return [];
 		}
 
-		return [
+		$data = [
 			'type' => 'link-text',
```

Comment:
This won't pass the validation (we should fix the schema if it does). When there is no link the type should be `line-text`.